modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -1652,6 +1652,8 @@ FeatureFloodCount::FeatureData::merge(FeatureData && rhs)
   // Logical AND here to combine flags (INACTIVE & INACTIVE == INACTIVE, all other combos are CLEAR)
   _status &= rhs._status;
 
+  _intersects_boundary = rhs._intersects_boundary;
```

Comment:
NO! This is definitely wrong. You are adding a line in the "merge" method. How do you combine data structures for two features, one that claims to intersect the boundary and one that does not. This would set the value in the combined feature to whatever is on the right. 

This is the correct logic:
`_intersects_boundary |= rhs._intersects_boundary;`